AEPMessaging/Sources/MessagingConstants.swift

```diff
@@ -58,7 +58,7 @@ enum MessagingConstants {
         enum Name {
             static let MESSAGE_INTERACTION = "Messaging interaction event"
             static let PUSH_NOTIFICATION_INTERACTION = "Push notification interaction event"
-            static let PUSH_PROFILE_EDGE = "Push notification profile edge event"
+            static let PUSH_PROFILE_EDGE = "Push notification token profile Edge event"
```

Comment:
Few Integration test and a unit test if failing. Probably because of this change.
Can we edit those tests to pass